vtex-sites/base.store
# Chore/updating analytics event schema

## What's the purpose of this pull request?
This PR updates the analytics event schema to match the new one from @faststore/sdk

## How it works? 
Read more [here](https://github.com/vtex/faststore/pull/1057)

## How to test it?
Check the window.dataLayer array to see if the events are being added properly to it

## References
https://github.com/vtex/faststore/pull/1057
